### DISCUSSION TITLE
Permission strategy for team with policy

### DISCUSSION THREAD

**Discussion by bluebill1049** [supabase/supabase#2960] [Questions] [answered]
Hi there, 

Is there a way we can define a policy for a team? right now my knowledge aware of how to set up a policy for authenticated users or rows that belong to the user. 

Would like to have UserA and UserB be able to read and write each other's data.

```
Team: 
----
A
B

User 
----
UserA TeamA (can read/write/delete UserB entries)
UserB TeamA (can read/write/delete UserA entries)
```

Thank you in advance. Loving Supabase ❤️

**Comment by silentworks** [comment, marked as answer]
You can check this existing discussion to see how to create more complex RLS policies.

https://github.com/supabase/supabase/discussions/811
